src/librustc_feature/active.rs

```diff
@@ -574,6 +574,9 @@ declare_features! (
     /// No longer treat an unsafe function as an unsafe block.
     (active, unsafe_block_in_unsafe_fn, "1.45.0", Some(71668), None),
 
+    /// Alloc calling `transmute` in const fn
+    (active, transmute_in_const_fn, "1.45.0", Some(53605), None),
```

Comment:
I've been trying to prefix all lang and lib features pertaining to const-eval with "const". Could we continue that by calling this `const_fn_transmute` or so?